client-flink/flink-1.14-shaded/pom.xml

```diff
@@ -82,9 +82,16 @@
                 <exclude>META-INF/*.DSA</exclude>
                 <exclude>META-INF/*.RSA</exclude>
                 <exclude>**/log4j.properties</exclude>
+                <exclude>META-INF/LICENSE.txt</exclude>
+                <exclude>META-INF/NOTICE.txt</exclude>
+                <exclude>LICENSE.txt</exclude>
+                <exclude>NOTICE.txt</exclude>
```

Comment:
@Apache9 the suggested `ApacheLicenseResourceTransformer` and `ApacheNoticeResourceTransformer` in [1] seem not to work as expected, I finally chose to filter those files out by rules.

[1] https://lists.apache.org/thread/8v4wy5o132rpsjync6465zztgjlf6h5p

Reply:
If there are conflict LICENSE file then the ApacheLicenseResourceTransformer will ignore them, but I think ApacheNoticeResourceTransformer will aggregate all the NOTICE files into one. Anyway, you can do it manually but the problem is once you add new dependencies, you still need to manually modify the NOTICE file.